pkg/api/api.go

```diff
@@ -176,6 +176,7 @@ func (a *API) RegisterAlertmanager(am *alertmanager.MultitenantAlertmanager, api
 	a.indexPage.AddLinks(defaultWeight, "Alertmanager", []IndexPageLink{
 		{Desc: "Status", Path: "/multitenant_alertmanager/status"},
 		{Desc: "Ring status", Path: "/multitenant_alertmanager/ring"},
+		{Desc: "Alertmanager", Path: "/alertmanager"},
```

Comment:
I wish there was a way to specify the tenant on this URL in a query param, instead of having to set up the header.

Reply:
Right. Still valuable for simple deployments with multi-tenancy disabled.

Reply:
I have multitenancy enabled on my personal instance, because I'm looking into the future, so this causes me a slight pain (need an extra ingress for this endpoint that sets the header)

Reply:
Sure, it's a pain, I agree. My comment was about the link.